sdk/identity/identity/src/msal/utils.ts

```diff
@@ -2,20 +2,26 @@
 // Licensed under the MIT license.
 
 import * as msalCommon from "@azure/msal-node";
+
 import { AccessToken, GetTokenOptions } from "@azure/core-auth";
 import { AuthenticationRecord, MsalAccountInfo, MsalResult, MsalToken } from "./types";
 import { AuthenticationRequiredError, CredentialUnavailableError } from "../errors";
-import { CredentialLogger, formatError, formatSuccess } from "../util/logging";
+import { CredentialLogger, credentialLogger, formatError, formatSuccess } from "../util/logging";
 import { DefaultAuthorityHost, DefaultTenantId } from "../constants";
+
 import { AbortError } from "@azure/abort-controller";
-import { MsalFlowOptions } from "./flows";
-import { isNode, randomUUID } from "@azure/core-util";
 import { AzureLogLevel } from "@azure/logger";
+import { isNode } from "@azure/core-util";
 
 export interface ILoggerCallback {
   (level: msalCommon.LogLevel, message: string, containsPii: boolean): void;
 }
 
+/**
+ * @internal
+ */
+const logger = credentialLogger("IdentityUtils");
```

Comment:
would love a refactor at some point to tease apart the browser/node divide in this file to be more fine-grained, there seems to be a lot of overlap today

Reply:
Yup! I'll create an issue for that 👍 there's probably a ton we can combine